packages/@tailwindcss-upgrade/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@tailwindcss/upgrade",
```

Comment:
Nit: The PR description mentions `@tailwindcss/codemod` and this here uses `@tailwindcss/upgrade` 😅